### DISCUSSION TITLE
posts/633

### DISCUSSION THREAD

**Discussion by giscus** [kakaotech/comments#16] [Announcements]
# posts/633

안녕하세요. 카카오 기술전략 소속의 로빈입니다.
현재 저는 카카오의 AI 교육 자...

https://tech.kakao.com/posts/633

**Comment by winterjung** [comment]
작성해주신 글 잘 읽었습니다. 한 가지 제보드릴 게 있는데 모바일 환경에서 터치영역에 따라 본문의 스크롤이 제대로 되지 않는 현상이 있어 알려드립니다.

https://github.com/user-attachments/assets/ad63b471-d39e-45a6-842a-049a06443619

**Reply by hooney** [reply]
버그 알려주셔서 감사합니다. 내부에서 빠르게 처리하도록 할게요. 감사합니다. 
재연을 위해, 확인 결과 아이폰 사파리/크롬 브라우저에서 발생하는 이슈로, 안드로이드폰 크롬 브라우저에서는 재연되지 않더군요.

**Reply by hooney** [reply]
@winterjung - 알려주신 버그 수정했습니다.

**Comment by GzuPark** [comment]
전반적인 정리해주셔서 감사합니다.
내용 중 오타가 있어 알려드립니다.

```diff
- Anthropic의 Claude 또한 버전 2에서 3으로 업그레이드되면서 Opus, Sonet 및 Janet 세 가지 모델을 선보였으며, 모두 이전 버전보다 성능이 크게 개선되었습니다.
+ Anthropic의 Claude 또한 버전 2에서 3으로 업그레이드되면서 Opus, Sonet 및 Haiku 세 가지 모델을 선보였으며, 모두 이전 버전보다 성능이 크게 개선되었습니다. 
```

**Reply by hooney** [reply]
오타 알려주셔서 감사합니다. 바로 수정했습니다.

**Comment by seochan99** [comment]
안녕하세요! 요즘 LLM에 관심이 많아 재밌고 유용하게 읽었습니다!

다만, `다양한 RAG(Reinforcement-Aware Generation) 기술들이 개발` 라고 언급해주시고 이후 `RAG`라는 약칭으로 사용하셨는데 이는 통상적으로 알려져있는 `RAG(Retrieval-Augmented Generation)`와 다른 개념으로 자주 활용되나요?
아니면 이 게시글에만 해당되는 내용일까요!?

다름이 아니라, 처음 보는 단어라 위 단어에 대해 서칭을 해봤는데 `Reinforcement-Aware`에 대해서 언급하는 곳이 이 게시물 밖에 없어서요! 

아무래도 앞뒤 문맥상 강화인지보다는 `검색증강(Retrieval-Augmented)`가 맞는 용어일듯한데, 제가 오개념을 알고 있나 싶어 댓글달았습니다!

**Reply by hooney** [reply]
@seochan99 오타 알려주셔서 감사합니다. 바로 수정했습니다.
